docs/releases/1.8.4.txt

```diff
@@ -14,3 +14,7 @@ Bugfixes
 
 * Added a system check warning if the old ``TEMPLATE_*`` settings are defined
   in addition to the new ``TEMPLATES`` setting.
+
+* Fixed `RawQuerySet` so `InvalidQuery` is not raised when using the
```

Comment:
single backticks should be doubled
Might be more useful to readers to change RawQuerySet to QuerySet.raw()